src/welcome-bare-metal.md

```diff
@@ -0,0 +1,21 @@
+# Welcome to Bare Metal Rust
+
+Today we will talk about 'bare-metal' Rust: running Rust code without an OS underneath us. This will
+be divided into several parts:
+
+ * What is `no_std` Rust?
+ * Writing firmware for microcontrollers.
+ * Writing bootloader / kernel code for application processors.
+ * Some useful crates for bare-metal Rust development.
+
+For the microcontroller part of the course we will use the [BBC micro:bit](https://microbit.org/) v2
+as an example. It's a development board based on the Nordic nRF51822 microcontroller.
```

Comment:
> Nordic nRF51822 microcontroller

This has no meaning to me :slightly_smiling_face:  Can it be described in more general terms, something like "a popular development board for getting started" or "a board with X, Y, and Z features"?

Reply:
Added some more details. I do expect people taking this course to know something about embedded development already though.